shopinvader_api_payment/routers/payment.py

```diff
@@ -59,6 +59,10 @@ def pay(
         payable_reference=payable_obj.payable_reference,
         amount=payable_obj.amount,
         currency_code=odoo_env["res.currency"].browse(payable_obj.currency_id).name,
+        amount_formatted=odoo_env["sale.order"]
+        .sudo()
+        .browse(payable_obj.payable_id)
+        .currency_id.format(payable_obj.amount),
```

Comment:
@sbidoul I've seen you removed `amount_formatted` from `Payable` object, I agree with you, but then we need to browse the record here only to retrieve the `amount_formatted`. And I think the `sudo()` is necessary as this route is public.
By keeping it in the `Payable` object we can avoid this browse.

Reply:
Yes, I think it's better to do a second little browse than putting thing in Payable that have conceptually nothing to do there.